### DISCUSSION TITLE
When I run a wasm program in container, it can not get environment variables.

### DISCUSSION THREAD

**Discussion by TaurusM** [WasmEdge/WasmEdge#1982] [Q&A]
Dockerfile like this:
![image](https://user-images.githubusercontent.com/28164162/196321380-031de422-a718-49e8-adbb-1216f625fc4a.png)

and some code about get environment variables:
![image](https://user-images.githubusercontent.com/28164162/196321395-3b6129ba-eb62-4ad2-b78a-ba8bbcba9d2a.png)

and environment variables truely exists:
![image](https://user-images.githubusercontent.com/28164162/196321429-d9952fe1-fd7c-4f86-833d-60e5ecfea8c7.png)

and the log is:
![image](https://user-images.githubusercontent.com/28164162/196321453-54b386b4-26a7-4d4a-ae62-d992617d0bd4.png)

the dependencies are:
![image](https://user-images.githubusercontent.com/28164162/196321468-8fdbe75b-058d-4d19-a641-5693f2fb01d9.png)

So, what is the problem, or I missed something important? Guys please help me.

**Comment by hydai** [comment]
Hi, this issue is fixed in crun. You can try the latest crun release to verify. Thanks.
